### ISSUE TITLE
Search button is invisible in Firefox

### ISSUE THREAD

**Issue by matthewpaulthomas** [canonical-websites/www.ubuntu.com#2341]
1\. In Firefox, open [ubuntu.com/](https://www.ubuntu.com/).
2\. Resize the window small enough that the “Menu” menu appears.
3\. Try to find the search function.

What happens: There is a search button, but it is invisible.

![2341-ubuntu com-search-button](https://user-images.githubusercontent.com/19801137/31884431-e5edd94a-b7e4-11e7-81b5-4c3de1c91835.png)

What should happen: The search button should be visible.

The same problem does not occur in Chrome.

---
*Reported from: https://www.ubuntu.com/

**Comment by matthewpaulthomas** [comment]
The button is `<div class="search-toggle u-hidden--large">`.

This has a background image applied to it: `background-image: url("data:image/svg+xml;utf8, <svg xmlns='http://www.w3.org/2000/svg' width='28' height='28' viewBox='0 0 90 90'><g color='#fff'><path fill='none' stroke-width='4' d='M0 0h90v90H0z'/><path d='M69 36.5a33 33.5 0 1 1-66 0 33 33.5 0 1 1 66 0z' transform='matrix(.636 0 0 .627 16.114 16.12)' fill='none' stroke='#fff' stroke-width='9.5'/><path d='M55.77 52.92L52.94 55.75l14 14 2.83-2.83-14-14z' fill='#fff' stroke-width='6' class='s0'/></g></svg>");`

Loading that `data:` URL in Firefox produces an error:
```
XML Parsing Error: unclosed token
Location: data:image/svg+xml;utf8,…
Line Number 1, Column 85:
 <svg xmlns='http://www.w3.org/2000/svg' width='28' height='28' viewBox='0 0 90 90'><g color='
------------------------------------------------------------------------------------^
```

**Comment by Caleb-Ellis** [comment]
I believe it's a simple encoding error. Replacing all instances of '#' with '%23' should fix the issue. I'll fix it alongside [#2326](https://github.com/canonical-websites/www.ubuntu.com/issues/2326) soon.

**Comment by matthewpaulthomas** [comment]
Aha, good catch. The SVG is valid but the URL is not. Verified that changing “#” to “%23” fixes the problem.

![2341-search-svg](https://user-images.githubusercontent.com/19801137/31932210-9a378b40-b89d-11e7-96af-1d51ff11ca1d.png)
